.coveragerc

```diff
@@ -0,0 +1,12 @@
+[run]
+source = __YOUR_PROJECT_FOLDER__
```

Comment:
is this literally supposed to be `__YOUR_PROJECT_FOLDER__` and not maybe `crawler` ? 

we'll see if it works, but, that just seems weird on a first look.